From 7eee42f5d268084171d435de2b16333d2a0f79ab Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 5 Aug 2015 11:31:52 +0300
Subject: [PATCH] journald: fix count of object meta fields

There are 12 object meta fields created in dispatch_message_real(), but
we only allocated space for 11. Fix this.

Fixes #866.

Cherry-picked from: 704e4fe7a18a13a8651c0064ef3eda91027baffc
Related: #1318994
---
 src/journal/journald-server.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/journal/journald-server.h b/src/journal/journald-server.h
index c96877c50..b1263a758 100644
--- a/src/journal/journald-server.h
+++ b/src/journal/journald-server.h
@@ -148,7 +148,7 @@ typedef struct Server {
 #define N_IOVEC_META_FIELDS 20
 #define N_IOVEC_KERNEL_FIELDS 64
 #define N_IOVEC_UDEV_FIELDS 32
-#define N_IOVEC_OBJECT_FIELDS 11
+#define N_IOVEC_OBJECT_FIELDS 12
 
 void server_dispatch_message(Server *s, struct iovec *iovec, unsigned n, unsigned m, const struct ucred *ucred, const struct timeval *tv, const char *label, size_t label_len, const char *unit_id, int priority, pid_t object_pid);
 void server_driver_message(Server *s, sd_id128_t message_id, const char *format, ...) _printf_(3,4);
